languages/csharp/reference/exercise-concepts/rna-trascriptions.md

```diff
@@ -0,0 +1,38 @@
+# RNA transcription
+
```

Comment:
Could you add a link to the implementation this is based on? See [this example](https://github.com/exercism/v3/blob/master/languages/csharp/reference/exercise-concepts/robot-name.md#concepts-of-robot-name).